src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements AreaCalculetion {

```

Comment:
The name `AreaCalculetion` should not be used for interfaces as it is a verb. Consider renaming it to something like `AreaCalculator` to follow Java naming conventions.